app/src/main/java/com/study/myapplication/feature/compare/model/CompareCoinInfo.kt

```diff
@@ -0,0 +1,10 @@
+package com.study.myapplication.feature.compare.model
+
+data class CompareCoinInfo(
+    var coinName: String? = null,
```

Comment:
```suggestion
    val coinName: String = "",
```
not null type을 사용하는게 좀 더 좋지 않을까요?

Reply:
6861f30
완료